src/openrct2-ui/input/MouseInput.cpp

```diff
@@ -1445,8 +1447,37 @@ void InputStateWidgetPressed(
                 STR_COLOUR_DARK_PINK_TIP,
                 STR_COLOUR_BRIGHT_PINK_TIP,
                 STR_COLOUR_LIGHT_PINK_TIP,
+                STR_COLOUR_DARK_OLIVE_DARK_TIP,
+                STR_COLOUR_DARK_OLIVE_LIGHT_TIP,
+                STR_COLOUR_SATURATED_BROWN_LIGHT_TIP,
+                STR_COLOUR_BORDEAUX_RED_DARK_TIP,
+                STR_COLOUR_BORDEAUX_RED_LIGHT_TIP,
+                STR_COLOUR_GRASS_GREEN_DARK_TIP,
+                STR_COLOUR_GRASS_GREEN_LIGHT_TIP,
+                STR_COLOUR_OLIVE_DARK_TIP,
+                STR_COLOUR_OLIVE_LIGHT_TIP,
+                STR_COLOUR_SATURATED_GREEN_LIGHT_TIP,
+                STR_COLOUR_TAN_DARK_TIP,
+                STR_COLOUR_TAN_LIGHT_TIP,
+                STR_COLOUR_DULL_PURPLE_LIGHT_TIP,
+                STR_COLOUR_DULL_GREEN_DARK_TIP,
+                STR_COLOUR_DULL_GREEN_LIGHT_TIP,
+                STR_COLOUR_SATURATED_PURPLE_DARK_TIP,
+                STR_COLOUR_SATURATED_PURPLE_LIGHT_TIP,
+                STR_COLOUR_ORANGE_LIGHT_TIP,
+                STR_COLOUR_AQUA_DARK_TIP,
+                STR_COLOUR_MAGENTA_LIGHT_TIP,
+                STR_COLOUR_DULL_BROWN_DARK_TIP,
+                STR_COLOUR_DULL_BROWN_LIGHT_TIP,
+                STR_COLOUR_INVISIBLE_TIP,
+                STR_COLOUR_VOID_TIP,
             };
-            WindowTooltipShow(OpenRCT2String{ _colourTooltips[dropdown_index], {} }, screenCoords);
+            auto stringId = COLOUR_UI_ORDER[dropdown_index];
```

Comment:
This is all levels of wrong. ColourUiOrder isn't a string id and none of this makes sense. Use a std::map instead of an array with this odd logic.